pkg/policy/mapstate.go

```diff
@@ -459,9 +460,9 @@ var visibilityDerivedFrom = labels.LabelArrayList{visibilityDerivedFromLabels}
 // L4-only key allowing the port is added.
 //
 // 'adds' and 'deletes' are updated with the changes made. 'adds' contains the new values
-// for both added and changed keys. 'deletes' contains the old values for both deleted and
-// changed keys.
-func (keys MapState) AddVisibilityKeys(e PolicyOwner, redirectPort uint16, visMeta *VisibilityMetadata, adds, deletes MapState) {
+// for both added and changed keys. 'deletes' contains the old values for changed keys.
+// This function does not delete any keys.
+func (keys MapState) AddVisibilityKeys(e PolicyOwner, redirectPort uint16, visMeta *VisibilityMetadata, adds Keys, oldValues MapState) {
```

Comment:
This code changes the named return value for `deletes`, please update the comment to reflect this change.

Reply:
Thanks, done!